src/networks/beta-1.md

```diff
@@ -0,0 +1,93 @@
+Unpacking and moving forc-run to /Users/sarah/.fuelup/toolchains/beta-1-aarch64-apple-darwin/bin
+Unpacking and moving forc-explore to /Users/sarah/.fuelup/toolchains/beta-1-aarch64-apple-darwin/bin
+Unpacking and moving forc-lsp to /Users/sarah/.fuelup/toolchains/beta-1-aarch64-apple-darwin/bin
+Unpacking and moving forc-fmt to /Users/sarah/.fuelup/toolchains/beta-1-aarch64-apple-darwin/bin
+Fetching core forc dependencies
+Installed forc v0.26.0 for toolchain 'beta-1-aarch64-apple-darwin'
+
+Adding component forc-wallet v0.1.2 to 'beta-1-aarch64-apple-darwin'
+Fetching binary from https://github.com/FuelLabs/forc-wallet/releases/download/v0.1.2/forc-wallet-0.1.2-aarch64-apple-darwin.tar.gz
+Unpacking and moving forc-wallet to /Users/sarah/.fuelup/toolchains/beta-1-aarch64-apple-darwin/bin
+Installed forc-wallet v0.1.2 for toolchain 'beta-1-aarch64-apple-darwin'
+
+Adding component fuel-core v0.10.1 to 'beta-1-aarch64-apple-darwin'
+Fetching binary from https://github.com/FuelLabs/fuel-core/releases/download/v0.10.1/fuel-core-0.10.1-aarch64-apple-darwin.tar.gz
+Unpacking and moving fuel-core to /Users/sarah/.fuelup/toolchains/beta-1-aarch64-apple-darwin/bin
+Installed fuel-core v0.10.1 for toolchain 'beta-1-aarch64-apple-darwin'
+
+Installed:
+- forc 0.26.0
+- forc-wallet 0.1.2
+- fuel-core 0.10.1
+
+
+The Fuel toolchain is installed and up to date
+```
+
+This installs the following components:
+
+- forc 0.26.0
+- forc-wallet 0.1.2
+- fuel-core 0.10.1
+
+If you wanted to create a custom toolchain with the same set of versions that get installed when you install the `beta-1 toolchain` you could do that manually. This is not recommended, as you can get this toolchain out of the box. To create a custom toolchain with the same versioning as the `beta-1` toolchain, run the following commands:
+
+```console
+$ fuelup toolchain new beta-1-custom-toolchain
+New toolchain initialized: beta-1-custom-toolchain
+default toolchain set to 'beta-1-custom-toolchain'
+$ fuelup component add forc@0.26.0
+Adding component forc v0.26.0 to 'beta-1-custom-toolchain'
+Fetching binary from https://github.com/FuelLabs/sway/releases/download/v0.26.0/forc-binaries-darwin_arm64.tar.gz
+Downloading component forc without verifying checksum
+Unpacking and moving forc to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-deploy to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-run to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-explore to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-lsp to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-fmt to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Fetching core forc dependencies
+Installed forc v0.26.0 for toolchain 'beta-1-custom-toolchain'
```

Comment:
Maybe we should only provide the commands not the outputs, but if you prefer with outputs we can change beta-2 equivalent of this as it does not have outptus